lib/websocket_server.rb

```diff
@@ -52,9 +52,6 @@ def initialize(options = {})
   end
 
   def call(env)
-    # Pass the request to ActionCable if it is for notifications
-    return ActionCable.server.call(env) if env['REQUEST_URI'].start_with?('/ws/notifications') && ::Settings.server.asynchronous_notifications
```

Comment:
verfiy:

This code no longer has notification (action cable) traffic going through here. so we no longer need to handle this case.

Reply:
:+1:

Reply:
Dumb question, are we still using `::Settings.server.asynchronous_notifications`?  At this point, is it even possible to disable async notifications?

Reply:
Yes, because that prevents the frontend to connect...

Reply:
Should we remove that option and assume it's on?  It doesn't have to be here but it seems wasteful to check if you can't operate the UI without async notifications.

Reply:
Well, you can ... if you turn it off, it would not complain, but you won't get realtime notifications.

Reply:
@jrafanie with other words, if notifications aren't available, the frontend will keep trying to connect every minute and throw ugly errors to the user. However, if `asynchronous_notifications` is set to false, the frontend will not try to connect.

Reply:
Great, if it makes sense, maybe we drop the `::Settings.server.asynchronous_notifications` completely in a followup PR.  I don't see how there's a way to deprecate it.  There is no alternative.

Reply:
Actually I'm still using that *feature*, it is really good for debugging if someone thinks that websockets break stuff in their code. It saved me multiple times :trollface: